cocos2d/core/components/CCButton.js

```diff
@@ -82,6 +82,7 @@ const State = cc.Enum({
  *  Button.EVENT_HOVER_IN<br/>
  *  Button.EVENT_HOVER_MOVE<br/>
  *  Button.EVENT_HOVER_OUT<br/>
+ * The user can get the current clicked object through the 'event.target' which is from the clickEvent callback function.
```

Comment:
User can get the current clicked node with 'event.target' from event object which is passed as parameter in the callback function of click event.